x-pack/plugin/esql/compute/gen/src/main/java/org/elasticsearch/compute/gen/ConvertEvaluatorImplementer.java

```diff
@@ -166,6 +166,9 @@ private MethodSpec evalVector() {
                 }
                 builder.endControlFlow();
                 builder.addStatement("nullsMask.set(p)");
+                if (resultType.equals(BYTES_REF)) {
+                    builder.addStatement("values.append(org.elasticsearch.compute.data.BytesRefBlockBuilder.NULL_VALUE)");
+                }
```

Comment:
This is the actual bug fix.

Reply:
I suggest something like
```
builder.addStatement("values.append($T.NULL_VALUE)", BYTES_REF_BLOCK_BUILDER);
```
which'll automatically import `BytesRefBlockBuilder` if it isn't in and leave the package off. It's just a tiny thing though.

Reply:
Thank you @nik9000. I was looking for a way to have this import done somehow.